coss/users/forms.py

```diff
@@ -1,17 +1,42 @@
 from django import forms
+from django.conf import settings
 from django.contrib.auth import get_user_model
 
 from wagtail.wagtailcore.models import Site
 from wagtail.wagtailusers.forms import UserCreationForm, UserEditForm
 
+from coss.base.mozillians import BadStatusCode, MozilliansClient, ResourceDoesNotExist
+
 
 UserModel = get_user_model()
 
 
+def fetch_mozillians_data(instance):
+    """Fetches profile data using the mozillians.org API."""
+
+    client = MozilliansClient(settings.MOZILLIANS_API_URL, settings.MOZILLIANS_API_KEY)
+    mozillian_user = None
+    try:
+        mozillian_user = client.lookup_user({'email': instance.email})
+    except (ResourceDoesNotExist, BadStatusCode):
+        pass
+
+    if mozillian_user:
+        instance.username = mozillian_user['username']
+        full_name = mozillian_user['full_name']['value']
+        instance.first_name, instance.last_name = full_name.split(' ', 1)
+        pass
+        if mozillian_user['photo']:
```

Comment:
``mozillian_user['photo']`` will always exist and will default to the default mozillians image.